xml/admin_ceph_upgrade.xml

```diff
@@ -543,15 +543,31 @@ Nodes running older software versions must be upgraded in the following order:
 <screen>&prompt.cephuser;ceph config set osd bluestore_fsck_quick_fix_on_mount true</screen>
```

Comment:
The above text notes that this command performs format conversion.  IMO this is not 100% correct. The command just enables that format conversion on the first start of an adopted OSD.